docs/content/concepts/terminology.md

```diff
@@ -1,100 +1,142 @@
-multiple different contexts, or for different use cases (a workspace that belongs to the user personally, or one that
-belongs to a business organization), the list of “all workspaces” itself needs to be exposed as an API object to an end
-user inside a workspace.  That workspace is “virtual” - it adapts or transforms the underlying source of truth for the
-object and potentially the schema the user sees.
+### Virtual Workspace
 
-## Index (e.g. Workspace Index)
+Virtual Workspaces are proxy-like API servers under a custom URL that provide a computed view of real Workspaces.
+That means that Virtual Workspaces provide Kubernetes-cluster-like HTTPs endpoints, just like classic Workspaces,
+so you can use the typical Kubernetes client tooling to interact with Virtual Workspaces. Moreover, Virtual Workspaces
+are the most often used for reconciling API objects exposed through the API binding mechanism (in a way that your
+controller will target the virtual workspace instead of each specific workspace).
 
-An index is the authoritative list of a particular API in their source of truth across the system.  For instance, in
-order for a user to see all the workspaces they have available, they must consult the workspace index to return a list
-of their workspaces.  It is expected that indices are suitable for consistent LIST/WATCHing (in the kubernetes sense) so
-that integrations can be built to view the list of those objects.
+An API object has one source of truth, i.e. it's transactional stored in one system. In the context of kcp, this
+means that an API object is stored in one workspace/logical cluster. However, a use case that arises is how can someone
+view API objects across different workspaces in a "single pane of glass" fashion. Concretely:
 
-Index in the control plane sense should not be confused with secondary indices (in the database sense), which may be
-used to enable a particular index.
+- the user may have access to different workspaces: a workspace that belongs to the user personally, or one that
+  belongs to a business organization. User might want to list and manage their API objects from all workspaces in a
+  single call (e.g. single `kubectl get` invocation),
+- Service Provider might want to get a list of all instances (from all users) of the API resource they provide, so that
+  they can do reconciliation, have insights into how their API is used, and more.
+
+More information, including concrete examples and a list of frequently asked questions, can be found in the
+[Virtual Workspaces document](../workspaces/virtual-workspaces/).
+
+## Exporting/Binding APIs
+
+One of the core values of the kcp project is to enable providing APIs that can be consumed by multiple workspaces.
+This is done via exporting/binding mechanism, i.e. via API Exports and API Bindings.
+
+The general workflow is:
+
+- Define your API resources similar to how you would define CustomResourceDefinitions (CRDs)
```

Comment:
```suggestion
- Define your API resources through APIResourceSchemes, very similar to how you would define CustomResourceDefinitions (CRDs)
```

Reply:
I actually had it exactly like that and decided to remove before pushing as I wasn't sure if it's too much for this document. 🙈 😅 I'll put it back in place like proposed.